biojava-core/src/main/java/org/biojava/nbio/core/sequence/template/AbstractSequence.java

```diff
@@ -323,6 +325,22 @@ public void setSequenceScore(Double sequenceScore) {
 	}
 
 	/**
+	 *
+	 * @return
+	 */
+	public List<AbstractReference> getReferences() {
```

Comment:
Hi, just committed the new changes. Do I need to close this request and create a new one? Thanks.